python/io.cpp

```diff
@@ -12,8 +12,8 @@ PYBIND11_MODULE(_io, m)
 {
   py::class_<Header>(m, "_header")
     .def_readonly("images_in_block", &Header::imagesInBlock)
-    .def_readonly("rows", &Header::rows)
```

Comment:
I don't think it matters lots, but I would assume the default is frame, and not prefix things referring to frames. I would instead prefix things referring to the larger thing, such as stemScanWidth, etc.

Reply:
I just wanted to make it clear, as it was confusing to me, but we an iterate.